devops/helm/crunchy-postgres/value-0c27fb-prod.yaml

```diff
@@ -0,0 +1,102 @@
+fullnameOverride: simsdb
```

Comment:
Is it necessary to have the namespace e.g `a6ef19` in it?

Reply:
we did the similar exercise for clamav, to have the namespaces values defined for a6ef19